### PR TITLE
Optimize sorting algorithm in AddPlaceCommand

### PR DESCRIPTION
**Pull request by tengkianen**

### CODE REVIEW

File: src/main/java/seedu/trippie/command/AddPlaceCommand.java [hunk 1]
```diff
@@ -105,15 +105,21 @@ public void execute(Ui ui, TrippieData trippieData) {
...
+                        && sortedPlaces.get(i).getPlaceStartTime() < sortedPlaces.get(j).getPlaceStartTime()) {
                     Collections.swap(sortedPlaces, i, j);
-                } else if (sortedPlaces.get(i).getPlaceDay() > sortedPlaces.get(j).getPlaceDay()) {
+                    swapped = true;
+                } else if (sortedPlaces.get(i).getPlaceDay() < sortedPlaces.get(j).getPlaceDay()) {
                     Collections.swap(sortedPlaces, i, j);
+                    swapped = true;
                 }
             }
+            if (swapped == false) {
```
Comment (felixhalim) [inline src/main/java/seedu/trippie/command/AddPlaceCommand.java:120]: Good, this way the sorting algorithm will terminate early :D

**Review by felixhalim — COMMENTED** [review]
Fix the checks, other than that it is good to go

File: src/main/java/seedu/trippie/command/AddPlaceCommand.java [hunk 2]
```diff
@@ -105,15 +105,21 @@ public void execute(Ui ui, TrippieData trippieData) {
...
                 if (sortedPlaces.get(i).getPlaceDay() == sortedPlaces.get(j).getPlaceDay()
-                        && sortedPlaces.get(i).getPlaceStartTime() > sortedPlaces.get(j).getPlaceStartTime()) {
+                        && sortedPlaces.get(i).getPlaceStartTime() < sortedPlaces.get(j).getPlaceStartTime()) {
                     Collections.swap(sortedPlaces, i, j);
-                } else if (sortedPlaces.get(i).getPlaceDay() > sortedPlaces.get(j).getPlaceDay()) {
+                    swapped = true;
+                } else if (sortedPlaces.get(i).getPlaceDay() < sortedPlaces.get(j).getPlaceDay()) {
                     Collections.swap(sortedPlaces, i, j);
+                    swapped = true;
                 }
```
Comment (ivanderjmw) [inline src/main/java/seedu/trippie/command/AddPlaceCommand.java:118]: Good use of bubble sort optimization.

**Review by ivanderjmw — APPROVED** [review]
LGTM.

**Pull request merged**